HOC/MotionWrapperCardDiv.tsx

```diff
@@ -12,13 +12,16 @@ const MotionWrapperCardDiv = ({
   delay,
   duration,
   additionalStyles = '',
+  once = true,
+  translateX,
 }: MotionWrapperCardDivProps) => (
   <motion.div
     variants={fadeIn(direction, type, delay, duration)}
-    viewport={{ once: false, amount: 0.1 }}
+    viewport={{ once, amount: 0.1 }}
     initial="hidden"
     whileInView="show"
-    className={additionalStyles}
+    className={`${additionalStyles} `}
+    whileHover={{ x: translateX }}
```

Comment:
You could also call this variable `className` to improve readability in usage. 
```
className={className}
```

That way using it is the same as any other element:
```js
<MotionWrapperCardDiv className="my classes here" />
```